Adyen/UI/Form/Items/FormImageItem/FormImageItem.swift

```diff
@@ -52,12 +55,10 @@ internal class FormImageView: FormItemView<FormImageItem> {
                                   compatibleWith: nil)
         imageView.translatesAutoresizingMaskIntoConstraints = false
         NSLayoutConstraint.activate([
-            self.heightAnchor.constraint(greaterThanOrEqualToConstant: item.size?.height ?? 46.0),
-            self.widthAnchor.constraint(greaterThanOrEqualToConstant: item.size?.width ?? 46.0),
-            imageView.heightAnchor.constraint(equalToConstant: item.size?.height ?? 46.0),
+            imageView.heightAnchor.constraint(equalToConstant: item.size.height),
             imageView.topAnchor.constraint(equalTo: layoutMarginsGuide.topAnchor, constant: 40.0),
```

Comment:
With this constraints FormImageView will always have height: 40 + item.size.height + 10.
Is this expected on a standalone component, when it is full screen?